amundsen_application/static/css/_list-group.scss

```diff
@@ -6,16 +6,14 @@
   .list-group-item {
     border-left: none;
     border-right: none;
-    cursor: pointer;
     padding: 0;
 
     &:hover {
-      border-color: $gray;
-      background-color: $gray-lightest;
-    }
-
-    &:hover + li {
-      border-top-color: $gray;
+      box-shadow: 0px 0px 1px 0px rgba(0, 0, 0, 0.12),0px 2px 3px 0px rgba(0, 0, 0, 0.16);
+      cursor: pointer;
+      // Workaround: The bottom border shadow is hidden by the item below it
+      // TODO: The hover style will render on top of the footer
+      z-index: 1;
```

Comment:
This is pretty much the same issue that we had when rearranging AvatarIcons on the frequent users section. In that case we ended up using incrementing z-index. 

Alternative Solutions:

1) Keep the box-shadow on the hovered element AND put an inset box-shadow on the next element. This way the shadow will appear on the last element as well.
```
&:hover {
  box-shadow: 1px 1px 1px 1px black;
}
&:hover + .list-group-item {
  box-shadow: inset 1px 1px 1px 1px black
}
```

2) Use `flex-direction: column-reverse` and render them in reverse order. Add a 'filler' element at the bottom with `flex-grow: 1; flex-shrink: 1` and give the other elements `flex-grow: 0; flex-shrink: 0;`... This might cause other issues.

Reply:
Thanks for the tip. Tried out Option 1 and that works well.